cfme/tests/configure/test_vmware_console_support.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.configure.configuration import VMwareConsoleSupport
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+
+
+@pytest.mark.tier(3)
+@pytest.mark.sauce
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+def test_vmware_console_support(appliance):
+    """Tests that the VMware Console Support setting may be changed."""
+    console_types = ['VMware VMRC Plugin', 'VNC', 'VMware WebMKS']
```

Comment:
**Required**
Add this as a class variable ('constant', all caps name) and use it to verify the parameter is right. Then in tests you can instead iterate over `VMwareConsoleSupport.CONSOLE_TYPES` (or whatever you call it).